tests/subsys/shell/shell_fuzzer/testcase.yaml

```diff
@@ -0,0 +1,6 @@
+tests:
+  shell.fuzzer:
+    tags: shell fuzzer
```

Comment:
it depends on the CONFIG_ARCH_POSIX_FUZZ_IRQ, so need add 
filter:  CONFIG_ARCH_POSIX_FUZZ_IRQ